src/components/Plots/AladinPlot.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div>
+    <div v-show="!aladinLoaded" :style="'width:' + plotWidth + '; height:' + plotHeight + ';'" class="text-center m-auto">
+      <i class="fa fa-spin fa-spinner" />
+    </div>
+    <div v-show="aladinLoaded" :id="plotId" :style="'width:' + plotWidth + '; height:' + plotHeight + ';'" class="m-auto"></div>
+  </div>
+</template>
+<script>
+import $ from 'jquery';
+
+export default {
+  name: 'AladinPlot',
+  props: {
+    // Unique ID for the plot
+    plotId: {
+      type: String,
+      required: true
+    },
+    plotWidth: {
+      type: String,
+      default: '400px'
+    },
+    plotHeight: {
+      type: String,
+      default: '400px'
+    }
+  },
+  data: function() {
+    return {
+      aladinLoaded: false
+    };
+  },
+  mounted: function() {
+    // Aladin does not come bundled up as an NPM package and instead much be included like this
```

Comment:
As this comment says, aladin is not available as an npm package, so it has to be loaded in like this, and then the `A`, which lets you create an aladin instance, is available globally. This is not ideal since that means `A` leaks out into all your components, but I didn't figure out a better way to do it, and hopefully `A` isn't all that common of a variable name so there won't be clashes.

Reply:
Good point... maybe I could make the url where aladin is available a property, point to u-strasbg.fr by default, and then in our lco frontend I could set it to use the aladin that we host in our CDN

Reply:
Yeah I'll do that